bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_rooglobalfunc.py

```diff
@@ -313,3 +313,37 @@ def DataError(etype):
             raise exception
 
     return RooFit._DataError(etype)
+
+
+def bindFunction(name, func, *variables):
+    """
+    Pythonize a Python-defined function.
```

Comment:
```suggestion
    Wrap an arbitrary function defined in Python or C++.
```
Since this is now the entry point for both the C++ and Python code path, the way it is implemented.